webapp/security/views.py

```diff
@@ -226,6 +226,47 @@ def _update_notice_object(notice, data):
     return notice
 
 
+def read_notice(notice_id):
+    """
+    GET method to get notice by id
+    """
+
+    if notice_id:
+        notice = db_session.query(Notice).get(notice_id)
+
+        if not notice:
+            return (
+                flask.jsonify(
+                    {"message": f"Notice {notice_id} does not exist"}
+                ),
+                400,
+            )
+
+        return flask.jsonify({"data": notice.as_dict()}), 200
+
+    limit = flask.request.args.get("limit", default=20, type=int)
+    offset = flask.request.args.get("offset", default=0, type=int)
+
+    notices = (
+        db_session.query(Notice)
+        .order_by(Notice.published)
+        .offset(offset)
+        .limit(limit)
+        .all()
+    )
+
+    return (
+        flask.jsonify(
+            {
+                "data": [n.as_dict() for n in notices],
```

Comment:
We tend to avoid one letter variable names, can we go with `notice` please?

Reply:
Changed. Also split the endpoint in two.